cmd/wallet/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/hashcloak/Meson-wallet-demo/pkg/ethers"
+	"github.com/hashcloak/Meson-client/pkg/client"
+	"github.com/katzenpost/client/config"
+)
+
+func main() {
+	cfgFile := flag.String("c", "katzenpost.toml", "Path to the server config file")
```

Comment:
Change the default from `katzenpost.toml` to `client.toml` for ease of use. Also, change the description as it's confusing.